Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx

```diff
@@ -283,6 +280,9 @@ export const ObiEditor: FC<ObiEditorProps> = ({
           selectedIds: [selected as string],
         });
         focused && onFocusSteps([focused], tab);
+
+        document.querySelector(`[${AttrNames.SelectedId}="${selected}"]`) &&
+          (document.querySelector(`[${AttrNames.SelectedId}="${selected}"]`) as Element).scrollIntoView(true);
```

Comment:
Here are two duplicated function calls, we can optimize it.
I suggest we extract `scrollIntoView` as a helper since we also need to leverage this helper in other place (e.g. recover focus state after refreshing).

```
function scrollNodeIntoView(nodeId: string) {
   const node: HTMLELement = queryNodeById(nodeId);
   node.scrollIntoView(true);
}
```